cmd/mergeRequest.go

```diff
@@ -6,13 +6,14 @@ import (
 
 // mrCmd represents the mr command
 var mergeRequestCmd = &cobra.Command{
-	Use:   "merge-request [remote]",
+	Use:   "merge-request [remote [branch]]",
 	Short: mrCreateCmd.Short,
 	Long:  mrCreateCmd.Long,
 	Args:  mrCreateCmd.Args,
 	Run:   runMRCreate,
 }
 
 func init() {
+	mergeRequestCmd.Flags().StringSliceP("message", "m", []string{}, "Use the given <msg>; multiple -m are concatenated as seperate paragraphs")
```

Comment:
Hi ! thanks for the feedback. I tried with `mergeRequestCmd.Flags().AddFlagSet(mrCreateCmd.Flags())` but I can't make it work. I am a rookie in GO though 😐could you try ? I created a branch https://github.com/Samox/lab/tree/fix-merge-request-cmd-NOTWORKING

Reply:
Got woken up by a train laying on the horn -- I may have given you a bad example. There's a working version in `cmd/snippetCreate.go`.

Reply:
Oh the problem is likely when it's running. You probably need to put the line in `cmd/mrCreate.go` after it adds the flags since the order or inits is essentially random here.

Reply:
Got it :) It is working